.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+	"image": "mcr.microsoft.com/devcontainers/universal:2",
+	"postCreateCommand": "cd /opt && wget 'https://github.com/wpilibsuite/vscode-wpilib/releases/download/v2023.2.1/vscode-wpilib-2023.2.1.vsix'",
+	"features": {
+	},
+	"customizations": {
+	  "vscode": {
+		"settings": {
+			"java.server.launchMode": "Standard",
+			"redhat.telemetry.enabled": true
+		},
+		"extensions": ["/opt/vscode-wpilib-2023.2.1.vsix","vscjava.vscode-java-pack","vscjava.vscode-gradle", "redhat.java", "vscjava.vscode-java-debug", "vscjava.vscode-java-dependency", "ms-vscode.cpptools"]
+	  }
+	}
+  }
```

Comment:
Nitpicking: would you mind adding an empty last line?

Reply:
I think this is what you meant